cmd/asciidoc.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var asciidocCmd = &cobra.Command{
+	Args:        cobra.ExactArgs(1),
+	Use:         "asciidoc [PATH]",
+	Aliases:     []string{"ad"},
```

Comment:
Since the file type is `.adoc` I suggest to change the alias from `ad` to `adoc`